rest_assured/testcases.py

```diff
@@ -50,28 +50,31 @@ def get_object(self, factory):
 
         return factory.create()
 
-    def setUp(self):
-        """Generates the main object and user instance if needed.
+    def get_user_factory(self):
```

Comment:
This isn't a getter method, it actually sets `user`. If we're to follow the code style properly this should be a pure getter, simply for allowing developers override this method.

So this should simply `getatter()` on `user_factory` and return it.